include/seqan3/argument_parser/detail/format_parse.hpp

```diff
@@ -163,11 +163,10 @@ class format_parse : public format_base
      * \tparam id_type The identifier type; must be either of type `char` if it denotes a short identifier or
      *                 std::string if it denotes a long identifier.
      * \param[in] begin_it The iterator where to start the search of the identifier.
-     *                     Note that the end iterator is kept as a member variable.
-     * \param[in] id       The identifier to search for (must not contain dashes).
-     * \returns An iterator pointing to the first occurrence of the identifier in
-     *          the list pointed to by begin_t. If the list does not contain the
-     *          identifier `id`, the member variable `end_of_options_it` is returned.
+     * \param[in] end_it The iterator where to end the search of the identifier.
```

Comment:
Isn't this "one past where you want the search to end"

Reply:
```suggestion
     * \param[in] end_it The iterator one past the end of where to search the identifier.
```